### PR TITLE
[OSDOCS-6598] Added information on what role is required for OADP installation on ROSA STS

### PR DESCRIPTION
**Pull request by fmcdonal**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-6598

Link to docs preview:
https://64045--docspreview.netlify.app/openshift-rosa/latest/rosa_backing_up_and_restoring_applications/backing-up-applications

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by fmcdonal** [comment]
PR: https://github.com/openshift/openshift-docs/pull/62016 for the same updates has been reviewed by QE and peer reviewed twice and Aaron agreed once steps have been replicated with QE he is happy. That PR https://github.com/openshift/openshift-docs/pull/62016 was merged but new requests for updates from this PR.

**Comment by mail2nadeem92** [comment]
Thanks for the PR :)

**Comment by fmcdonal** [comment]
@mail2nadeem92 please approve at your convenience for merging.

**Comment by fmcdonal** [comment]
@yuokada126 feel free to review also, thanks Frances

**Comment by mnadeem92** [comment]
@fmcdonal  +1 
It looks good to me and I have verified all the comments including one that we discussed today are implemented.
Thanks for working on it. 👍

**Comment by fmcdonal** [comment]
Spoke to @YUki and advised to go ahead with the merge review

**Comment by fmcdonal** [comment]
@opayne1 This has been peer reviewed twice on: https://github.com/openshift/openshift-docs/pull/62016 and it should not have been merged but it was so I discussed it with our team and it did not need another peer review as changes were within the code tath was in inaccurate

File: modules/oadp-installing-oadp-rosa-sts.adoc [hunk 1]
```diff
@@ -34,6 +34,13 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 EOF
 ----
 
+.. Create a namespace for OADP:
++
+[source, terminal]
```
Comment (jeana-redhat) [inline modules/oadp-installing-oadp-rosa-sts.adoc:39]: This seems to be fine in the preview build, but I am not sure the portal transform will like it.
```suggestion
[source,terminal]
```

**Review by jeana-redhat — COMMENTED** [review]
One small format thing, and the commits need to be squashed. Please add back to the merge review queue when ready.

**Comment by jeana-redhat** [comment]
Looks like some leftover rebase stuff got in here, throughout the files. Look for `<<<<<<<` and `>>>>>>>`

**Comment by fmcdonal** [comment]
thanks @adellape

**Pull request merged**